lib/config.js

```diff
@@ -35,6 +36,20 @@ exports.get = () => {
   if (errors.length > 0) {
     throw new Error('Error in .tldrrc configuration:\n' + errors.join('\n'));
   }
+
+  // Setting correct languages
+  merged['languages'] = ['en'];
+  // Get the primary & secondary language.
+  let langs = utils.localeToLang(process.env.LANG);
+  merged['languages'] = merged['languages'].concat(langs);
+  
+  if(process.env.LANGUAGE !== undefined) {
+    let langs = process.env.LANGUAGE.split(':');
+    for(let lang of langs) {
+      merged['languages'] = merged['languages'].concat(utils.localeToLang(lang));
+    }
```

Comment:
Similar as above, can use `map` to make this more succinct:

```suggestion
    merged.languages.push(...langs.map((lang) => utils.localeToLang(lang)));
```

The maximum number of languages is also low enough that doing a spread to push vs individually will have no practical difference in terms of speed/memory.